### PR TITLE
v1.1.0: Poggit rules compatible + minor improvements

### PR DESCRIPTION
**Pull request by matcracker**
Now the library is compatible with poggit rules. On the occasion, some improvements were made.

### CODE REVIEW

File: src/JackMD/UpdateNotifier/task/UpdateNotifyTask.php [hunk 1]
```diff
@@ -73,28 +71,24 @@ public function onRun(): void{
...
-	
-	/**
-	 * @param Server $server
-	 */
-	public function onCompletion(Server $server): void{
-		$pluginName = $this->pluginName;
-		$plugin = $server->getPluginManager()->getPlugin($pluginName);
+
+	public function onCompletion(Server $server) : void{
+		$plugin = Server::getInstance()->getPluginManager()->getPlugin($this->pluginName);
```
Comment (matcracker) [inline src/JackMD/UpdateNotifier/task/UpdateNotifyTask.php:79]: This is already ready for PM4.

Comment (JackMD) [reply src/JackMD/UpdateNotifier/task/UpdateNotifyTask.php:79]: But in 4.0 (last time i saw) the onCompletion() method didnt have any params?

**Comment by matcracker** [comment]
Yes, you just need to remove the param to support the 4.0

File: src/JackMD/UpdateNotifier/task/UpdateNotifyTask.php (hunk 1 above)
Comment (matcracker) [reply src/JackMD/UpdateNotifier/task/UpdateNotifyTask.php:79]: Yes, you just need to remove the param to support the 4.0

**Pull request merged**